Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -387,13 +387,21 @@ public function __construct()
 
         $installedPhpVersion = phpversion();
 
+        $composerFileContents = file_get_contents(__DIR__.'/../composer.json');
+        if (array_key_exists('php', $composerFileContents['require'])) {
+            $phpDependency = $composerFileContents['require']['php'];
```

Comment:
Yes, it needs to be based on the Symfony version. The required PHP version in composer.json is user configurable.